src/components/UI/LabelsList.tsx

```diff
@@ -2,22 +2,32 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import ValueLabel from 'UI/ValueLabel';
 
+interface ILabel {
```

Comment:
I noticed you're duplicating this in multiple places. Do you mind storing it in a common place (such as model, or maybe it is already in the js client, and you can import it from there)?

Reply:
I'd say the best place for it is in the js client, as we'll probably use all the types in there when we'll migrate our redux logic to typescript